Loop/Managers/DeviceDataManager.swift

```diff
@@ -1137,6 +1137,10 @@ extension DeviceDataManager: PumpManagerOnboardingDelegate {
             self.settingsManager.storeSettings()
         }
     }
+
+    func pumpManagerOnboarding(didPauseOnboarding pumpManager: PumpManagerUI) {
+        
+    }
```

Comment:
empty on purpose?

Reply:
Yes, this is for when DeviceDataManager is hosting the pump onboarding (vs an onboarding plugin hosting it). This is not exercised in Tidepool Loop (or it shouldn't be; before this commit, there were paths where it could be). There may need to eventually be a modal dialog dismiss here for DIY, but I haven't tested it yet.

Reply:
[nit] maybe add a comment? Or make it optional.